app/src/main/java/com/framgia/wsm/utils/validator/Validator.java

```diff
@@ -7,7 +7,7 @@
 import android.text.TextUtils;
 import android.util.Log;
 import android.util.SparseArray;
-import com.framgia.wsm.screen.BaseViewModel;
+import com.framgia.wsm.screen.changepassword.ChangePasswordViewModel;
```

Comment:
ERROR: Unused import: com.framgia.wsm.screen.changepassword.ChangePasswordViewModel. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports